content/rules/bridges.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Bridges
+---
+
+# Bridges
+
+Problem: Not all the people or organizations you follow may have migrated from Twitter to open social systems like Mastodon, or they moved to a different decentralized network like Bluesky or Nostr which do not use ActivityPub as their federation protocol.
+
+One option is to follow such accounts on other networks using a bridge.
+
+There are a few issues here:
+
+- While some people may intend to follow accounts from another network through a bridge and understand the limitations, these accounts end up federated and searchable by anyone else in the instance.
+- These bridges may not respect the original author's privacy, or require them to opt-out of federation, republishing without the original author's permission or consent.
+- There are a variety of backend issues when these bridged accounts are followed, unfollowed, replied to, and so on, and their implementations don’t respond properly because they're overloaded or incorrectly implemented.
+
+In general, vmst.io does not connect to these services or allow our user's posts to be accessed through these services.
+
+## Twitter
+
+Bridging with Twitter is typically a one-way operation, where software like [bird.makeup](https://sr.ht/~cloutier/bird.makeup/) running on a server connecting to the Twitter API and creates stub accounts of Twitter users, to then mirror their content as accounts on Mastodon.
+
+Although the bridge might work to make Twitter content visible, it can't create real conversation; your likes and replies with the person running the Twitter account are not sent back to Twitter and are never seen by the author.
```

Comment:
Great use of a semicolon, but readers are almost always better served by just splitting the sentence in two.

```suggestion
Although the bridge might work to make Twitter content visible, it can't create real conversation.
Your likes and replies with the person running the Twitter account are not sent back to Twitter and are never seen by the author.
```